devnet-sdk/system/wallet.go

```diff
@@ -34,7 +35,19 @@ type wallet struct {
 	chain      internalChain
 }
 
-func newWallet(pk string, addr types.Address, chain *chain) (*wallet, error) {
+func newWalletMapFromDescriptorWalletMap(descriptorWalletMap descriptors.WalletMap, chain internalChain) (map[string]Wallet, error) {
```

Comment:
We can use `type WalletMap map[string]Wallet` here.

Reply:
Fixed in 2895f546